core/src/main/kotlin/entity/interaction/GuildInteraction.kt

```diff
@@ -12,11 +12,20 @@ public sealed interface GuildInteraction : Interaction, GuildInteractionBehavior
 
     override val guildId: Snowflake get() = data.guildId.value!!
 
-    /** Overridden permissions of the interaction [invoker][user] in the [channel][GuildInteractionBehavior.channel]. */
+    /**
+     * [Permissions] the [interaction invoker][user] has within the [channel][GuildInteractionBehavior.channel] the
+     * interaction was sent from.
+     */
     public val permissions: Permissions get() = data.permissions.value!!
```

Comment:
Should we rename this to `userPermissions` or `memberPermissions`? That way it would be harder to confuse it with `appPermissions`.

Reply:
`app` tells the difference between both doesn't it